cmd/server/pactasrv/blobs.go

```diff
@@ -0,0 +1,112 @@
+package pactasrv
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/RMI/pacta/db"
+	"github.com/RMI/pacta/oapierr"
+	api "github.com/RMI/pacta/openapi/pacta"
+	"github.com/RMI/pacta/pacta"
+	"go.uber.org/zap"
+)
+
+func (s *Server) AccessBlobContent(ctx context.Context, request api.AccessBlobContentRequestObject) (api.AccessBlobContentResponseObject, error) {
+	actorOwnerID, err := s.getUserOwnerID(ctx)
+	if err != nil {
+		return nil, err
+	}
+	actorUserID, err := getUserID(ctx)
+	if err != nil {
+		return nil, err
+	}
+	actorIsAdmin, actorIsSuperAdmin, err := s.isAdminOrSuperAdmin(ctx)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Depending on how popular needing this set of info is, I'd add a "gimme the deets" helper that returns all this stuff in a nice `UserInfo` struct or similar.

Reply:
Great suggestion - done.